### PR TITLE
File reorganization

### PR DESCRIPTION
**Pull request by grugnog**

### CODE REVIEW

File: common-practices-tools/security/contingency-plan.md [hunk 1]
```diff
@@ -75,21 +75,21 @@ The IC first notifies and coordinates with the people who are authorized to deci
...
+The IC keeps a log of the situation in the [`#general`](https://civicactions.slack.com/messages/general/) Slack channel or within a client-specific Slack channel, JIRA ticket, or GitHub issue. If this is also a security incident, the IC also follows the [security incident communications process](incident-response-plan.md#3-initiate-the-response#3-initiate-the-response). The IC should delegate assistant ICs for aspects of the situation as necessary.
 
 ### Recovery
 
 The Incident Response Team assesses the situation and works to recover the system. See the list of [external dependencies](#external-dependencies) for procedures for recovery from problems with external services.
 
-If this is also a security incident, the IC also follows the [security incident assessment](incident-response-plan.md#4-assess-the-incident) and [remediation](incident-response-plan.md#5-remediate) processes.
+If this is also a security incident, the IC also follows the [security incident assessment](incident-response-plan.md#4-assess-the-incident#4-assess-the-incident) and [remediation](incident-response-plan.md#5-remediate#5-remediate) processes.
```
Comment (dmundra) [inline common-practices-tools/security/contingency-plan.md]: Extra '#' anchor.

File: about-this-guidebook/guidebook-governance.md [hunk 2]
```diff
@@ -12,7 +12,7 @@ These teams are listed on GitHub as [subteams](https://github.com/orgs/CivicActi
 
 | **Directory in `docs/`**                                        | **Subteam (Working group / Practice Area)**                                    |
```
Comment (dmundra) [inline about-this-guidebook/guidebook-governance.md]: We should update the folder names here in future pull requests.

File: common-practices-tools/security/contingency-plan.md [hunk 3]
```diff
@@ -75,21 +75,21 @@ The IC first notifies and coordinates with the people who are authorized to deci
...
 
 If the IC assesses that the overall response process is likely to last longer than 3 hours, the IC should organize shifts so that each responder works on response for no longer than 3 hours at a time, including handing off their own responsibility to a new IC after 3 hours.
 
 ### Reconstitution
 
 The Incident Response Team tests and validates the system as operational.
 
-The Incident Commander declares that recovery efforts are complete and notifies all relevant people. The last step is to schedule a postmortem to discuss the event. This is the same as the [security incident retrospective process](incident-response-plan.md#conducting-a-retrospective).
+The Incident Commander declares that recovery efforts are complete and notifies all relevant people. The last step is to schedule a postmortem to discuss the event. This is the same as the [security incident retrospective process](incident-response-plan.md#conducting-a-retrospective#conducting-a-retrospective).
```
Comment (dmundra) [inline common-practices-tools/security/contingency-plan.md]: Extra '#' anchor.

File: practice-areas/engineering/tech-lead.md [hunk 4]
```diff
@@ -182,6 +182,6 @@ Yes. Ultimately this project role (an individual or multiple team members) is gi
 - [#engineering-techlead slack channel](https://civicactions.slack.com/archives/C017JL86MAM) (review the pinned messages).
 - [Tech Lead Reading List](https://docs.google.com/spreadsheets/d/1QfK1-7IOqb7_N5451Y0FNiwvDlxirETVnl71sB_DUXE/edit#gid=0).
 - [Tech Lead Call Agendas, Resources, and Notes](https://docs.google.com/document/d/1abRkMOuB9Kb2n8jlzjJnEClPCbAGOtZ6xclo6UlLiag/edit#heading=h.bsicsupyvxmj).
-- [Onboarding a New Project Team Member](../070-project-management/onboarding-new-project-team-member/#onboarding-a-new-project-team-member).
-- [Discovery Services](../080-sales-and-marketing/service-catalog/discovery-services/).
-- [The Sales Proposal Process](../080-sales-and-marketing/sales-proposal-process/#3-tech-lead).
+- [Onboarding a New Project Team Member](../../070-project-management/onboarding-new-project-team-member/#onboarding-a-new-project-team-member#onboarding-a-new-project-team-member).
+- [Discovery Services](../../080-sales-and-marketing/service-catalog/discovery-services).
+- [The Sales Proposal Process](../../080-sales-and-marketing/sales-proposal-process/#3-tech-lead#3-tech-lead).
```
Comment (dmundra) [inline practice-areas/engineering/tech-lead.md]: The following links will need some fixing since they reference old ones.

Comment (dmundra) [reply practice-areas/engineering/tech-lead.md]: Might just need to be deleted.

**Review by dmundra — APPROVED** [review]
@grugnog this looks great. I noticed and left comments for some extra anchors being added, the guidebook governance page will need updating (I will create a ticket about), and one page that will need links updated.

File: about-this-guidebook/guidebook-governance.md (hunk 2 above)
Comment (grugnog) [reply about-this-guidebook/guidebook-governance.md]: I just removed the table for now - I think CODEOWNERS is a better way to manage these associations.

Comment (dmundra) [reply about-this-guidebook/guidebook-governance.md]: Agreed!

**Pull request merged**